### PR TITLE
SignedJWT

### LINKED ISSUE #163: Question: Signed JWT
Hello,

Just wanted to raise a question and hopefully you can help. I am looking at Signed JWT feature of Keycloak as described here:

- https://www.keycloak.org/docs/latest/server_admin/#_client-credentials
- https://www.janua.fr/understanding-client-authenticator-security-with-keycloak/

Is this supported or in the pipeline?

### PR DESCRIPTION
**Pull request by SVilgelm**
Fix #163

### CODE REVIEW

**Comment by SVilgelm** [comment]
@alejandro-falkowski-gelato Please check this PR. Is it what you need?

@Nerzal please review

File: pkg/jwx/jwx.go [hunk 1]
```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+// SignClaims sings the given claims using a given key and a method
```
Comment (alejandro-falkowski-gelato) [inline pkg/jwx/jwx.go:17]: `signs`

**Comment by alejandro-falkowski-gelato** [comment]
> @alejandro-falkowski-gelato Please check this PR. Is it what you need?
> 
> @Nerzal please review

Looks good from my side, thanks a lot!

**Comment by SVilgelm** [comment]
@alejandro-falkowski-gelato thank you, I'm not sure that the `LoginClientSignedJWT` is really useful, but it can be used as an example of how to prepare a payload for the `GetToken` function

**Comment by alejandro-falkowski-gelato** [comment]
> @alejandro-falkowski-gelato thank you, I'm not sure that the `LoginClientSignedJWT` is really useful, but it can be used as an example of how to prepare a payload for the `GetToken` function

OK great I will try it out when I get a chance.

**Comment by SVilgelm** [comment]
@Nerzal please review

**Comment by SVilgelm** [comment]
Released in v6.2.0

**Pull request merged**